esmvaltool/utils/cmorizers/obs/utilities.py

```diff
@@ -18,9 +21,24 @@ def _read_cmor_config(cmor_config):
         os.path.dirname(__file__), 'cmor_config', cmor_config)
     with open(reg_path, 'r') as file:
         cfg = yaml.safe_load(file)
+    timestamp = datetime.datetime.utcnow()
+    timestamp_format = "%Y-%m-%d %H:%M:%S"
+    now_time = timestamp.strftime(timestamp_format)
+    cfg['proj']['metadata_attributes']['CMORcreated'] = now_time
+    cfg['proj']['cmip_table'] = CMOR_TABLES[cfg['proj']['project']]
```

Comment:
```suggestion
    cfg['proj']['cmor_table'] = CMOR_TABLES[cfg['proj']['project']]
```
is possibly more clear

Reply:
Ok. This was the easiest part!